launch_ros/launch_ros/utilities/evaluate_parameters.py

```diff
@@ -67,7 +67,9 @@ def check_sequence_type_is_allowed(sequence):
             if isinstance(value[0], Substitution):
                 # Value is a list of substitutions, so perform them to make a string
                 evaluated_value = perform_substitutions(context, list(value))
-                yaml_evaluated_value = yaml.safe_load(evaluated_value)
+                yaml_evaluated_value = yaml.load(evaluated_value)
```

Comment:
why stop using safe load? I don't think that's a good idea.

Reply:
That's a mistake. A leftover of a previous attempt using a custom Loader for a different reason. Good catch!